### PR TITLE
MF-487:Success message for patient registration

### PR DESCRIPTION
**Pull request by tejasdeshpande100**
Issue: [MF-487](https://issues.openmrs.org/projects/MF/issues/MF-487?filter=allopenissues&orderby=created+DESC%2C+priority+DESC%2C+updated+DESC)
Now when the user hits the Create patient button/ Update patient button after filling all the other fields, the user should be able to see a success message confirming that the new patient has been registered into the system and can be searched either using their names or their ID.
However as you can see in the [design](https://app.zeplin.io/project/5f7223cfda10f94d67cec6d0/screen/5f896066713ff75e528c3ccc), the toast notification should be at the top right corner and with a green background. I guess is currently not possible until [MF-448](https://issues.openmrs.org/browse/MF-448?src=confmacro) issue is resolved as the showToast function needs some work,
after which this PR should do exactly what is required.

### CODE REVIEW

File: src/patient-registration/patient-registration.component.tsx [hunk 1]
```diff
@@ -6,6 +6,7 @@ import camelCase from 'lodash-es/camelCase';
 import capitalize from 'lodash-es/capitalize';
 import find from 'lodash-es/find';
 import Button from 'carbon-components-react/es/components/Button';
+
```
Comment (FlorianRappl) [inline src/patient-registration/patient-registration.component.tsx]: Why insert this line?

File: src/patient-registration/patient-registration.component.tsx [hunk 2]
```diff
@@ -503,6 +504,17 @@ export const PatientRegistration: React.FC = () => {
           results.then(response => {}).catch(err => {});
 
           navigate({ to: getAfterUrl(response.data.uuid) });
+          existingPatient
+            ? showToast({
+                description: "The patient's information has been successfully updated",
```
Comment (FlorianRappl) [inline src/patient-registration/patient-registration.component.tsx]: Is this taken from the design?

If yes, ok - but we need all these strings translated. If no - I'd argue to drop it.

File: src/patient-registration/patient-registration.component.tsx (hunk 1 above)
Comment (tejasdeshpande100) [reply src/patient-registration/patient-registration.component.tsx]: Oops, I will get rid of it.

File: src/patient-registration/patient-registration.component.tsx (hunk 2 above)
Comment (tejasdeshpande100) [reply src/patient-registration/patient-registration.component.tsx]: Yes it is also from the designs, I will get it translated.

File: src/patient-registration/patient-registration.component.tsx [hunk 3]
```diff
@@ -503,6 +503,17 @@ export const PatientRegistration: React.FC = () => {
           results.then(response => {}).catch(err => {});
 
           navigate({ to: getAfterUrl(response.data.uuid) });
+          existingPatient
+            ? showToast({
+                description: t("The patient's information has been successfully updated"),
```
Comment (FlorianRappl) [inline src/patient-registration/patient-registration.component.tsx]: Can we get proper locale keys such as `updatePatientSuccessDescription`, `updatePatientSuccessTitle` etc.

**Comment by tejasdeshpande100** [comment]
I am closing this pull request since It got a little messy, I have created a new one: #93  which should serve the purpose.

**Pull request closed without merging**